app/views/rating/applications/index.csv.slim

```diff
@@ -10,12 +10,15 @@
       'Voluneer',
       'VolunteerHours',
       'Flags',
-      'Picks1',
-      'Picks2',
-      'Likes1',
-      'Likes2',
+      'Signoff Project1',
+      'Signoff Project2',
+      'Fav Project1',
+      'Fav Project1',
+      'Picks',
+      'Likes',
       'AveragePoints',
-      'Ratings Misc'],
```

Comment:
😂  oh hell, I already thought it must be something like this. I originally wrote a `%w()` style array but did not know how to use this in slim 🙈